internal/page/donor/upload_evidence.go

```diff
@@ -97,39 +102,61 @@ func readUploadEvidenceForm(r *http.Request) *uploadEvidenceForm {
 	}
 
 	// upload part
-	part, err = reader.NextPart()
-	if err != nil && err != io.EOF {
-		return &uploadEvidenceForm{Error: err}
-	}
+	var files []File
```

Comment:
we should limit the number of files that can be attached, say 5 as a number to start with?

Reply:
5 seems reasonable 👍